src/main/utility/utility.c

```diff
@@ -356,3 +356,16 @@ struct timespec utility_timespecFromMillis(int64_t millis) {
         .tv_nsec = (millis % 1000) * 1000000, // ms to ns
     };
 }
+
+
+GString* utility_getFileContent(FILE *stream) {
+    char buf[BUFSIZ];
+    GString *content = g_string_new(NULL);
+
+    memset(buf, 0, sizeof(buf));
+    while (NULL != fgets(buf, sizeof(buf), stream)) {
+        g_string_append(content, buf);
+    }
+
+    return content;
```

Comment:
This string is still empty.

Reply:
In which case ? I didn't obtain this kind of error in my tests.

Reply:
Ah I'd misread the code. This implementation does look correct, though I'd still lean towards just using "/dev/stdin" to the existing helper rather than adding a new one.